### PR TITLE
added quick fix and code actions on save docs

### PR DESCRIPTION
**Pull request by justschen**
- nearby quick fix documentation added in various places/languages
- code actions on save for typescript and javascript

cc. @gregvanl

### CODE REVIEW

**Review by hooooh2 — APPROVED** [review]
https://apps.apple.com/app/id1508990688

File: docs/languages/javascript.md [hunk 1]
```diff
@@ -219,13 +223,18 @@ Organize imports can also be done automatically when you save a JavaScript file
 The `editor.codeActionsOnSave` setting lets you configure a set of Code Actions that are run when a file is saved. For example, you can enable organize imports on save by setting:
 
 ```json
```
Comment (rzhao271) [inline docs/languages/javascript.md:225]: Check if there's jsonc used elsewhere for the site. It might render the comments properly.

Comment (justschen) [reply docs/languages/javascript.md:225]: fixed ! was because it was updated elsewhere i believe

**Review by rzhao271 — APPROVED** [review]

File: docs/languages/javascript.md (hunk 1 above)
Comment (rzhao271) [reply docs/languages/javascript.md:225]: If it's updated elsewhere, that sounds like it was an intentional change.
It looks like it's still json after the merge conflict fix, but we can leave it as-is.

**Pull request merged**